migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```

Comment:
Here and in some other places: is it actually necessary to have foreign code in our repo? Is there a better way of handling this? 